app/views/CheckYourClaimView.scala.html

```diff
@@ -73,7 +73,7 @@ <h1 class="govuk-heading-xl">@messages("checkYourClaimView.heading")</h1>
     <h2 class="govuk-heading-m">@messages("checkYourClaimView.subheading.1")</h2>
 
     <p class="govuk-body">@messages("checkYourClaimView.text.1.1")
-        <span class="govuk-!-font-weight-bold">&pound;@appConfig.taxReliefPerWeek2020</span>
+        <span class="govuk-!-font-weight-bold">&pound;@appConfig.taxReliefPerWeek(CurrentYear)</span>
         @messages("checkYourClaimView.text.1.2")
     </p>
```

Comment:
We want to show a full breakdown if claim amounts, so we should group the claimItems by their tax relief per week and duplicate this pair of paragraph + list of items for each unique claim amount